code/run_all.py

```diff
@@ -0,0 +1,106 @@
+import config
```

Comment:
i don't get why this exists if the `.sh` file is there

Reply:
@jasonwei20 @ntomita This was left from my testing on Windows.  It should be removed for now.

Reply:
@jasonwei20 @ntomita I ended up leaving the file in because the user cannot change the `argparse` flags if they use `run_all.sh`.

Reply:
Cant they just edit the run_all file? This duplicated code is hard to maintain though, and in general, duplicated code is bad.

Reply:
@JosephDiPalma Duplicated code is my main concern here. Can we actually call each numbered file with args here rather than reimplementing each process, maybe by using subprocess or os.system call?